src/taco-dependency-installer/installers/androidSdkInstaller.ts

```diff
@@ -77,10 +67,6 @@ class AndroidSdkInstaller extends InstallerBase {
             });
     }
 
-    protected postInstallWin32(): Q.Promise<any> {
-        return this.postInstallDefault();
-    }
-
     protected downloadDarwin(): Q.Promise<any> {
```

Comment:
All these methods, downloadDarwin(), downloadWin32(), updateVariablesWin32(), updateVariablesDarwin(), etc, would go in the specific implementation instead of living in the same class.